pandas/core/indexes/datetimelike.py

```diff
@@ -903,6 +926,15 @@ def _is_convertible_to_index_for_join(cls, other: Index) -> bool:
             return True
         return False
 
+    def _wrap_joined_index(self, joined: np.ndarray, other):
```

Comment:
why is this added?

Reply:
My bad I failed my rebase, this is old code I missed.